tick/linear_model/tests/serializing_test.py

```diff
@@ -0,0 +1,49 @@
+
+# License: BSD 3 clause
+
+
+import io, unittest
+import numpy as np
+
+import dill as pickle
+
+from tick.solver.tests import TestSolver
+
+from tick.prox import ProxL1
+from tick.linear_model import ModelLinReg, SimuLinReg
+from tick.linear_model import ModelLogReg, SimuLogReg
+from tick.linear_model import ModelPoisReg, SimuPoisReg
+from tick.linear_model import ModelHinge, ModelQuadraticHinge, ModelSmoothedHinge
+
+from tick.simulation import weights_sparse_gauss
+
+class Test(TestSolver):
+    def test_solver_saga(self):
+
+        model_map = {
+            ModelLinReg : SimuLinReg,
+            ModelLogReg : SimuLogReg,
+            ModelPoisReg: SimuPoisReg,
+            ModelHinge  : SimuLogReg,
+            ModelQuadraticHinge : SimuLogReg,
+            ModelSmoothedHinge : SimuLogReg,
+        }
+
+        for mod in model_map:
+            model = mod(fit_intercept=False)
+
+            coeffs0 = weights_sparse_gauss(20, nnz=5)
+            interc0 = None
+
+            features, labels = model_map[mod](
+                coeffs0, interc0, n_samples=3000, verbose=False,
```

Comment:
Is this faster with `n_samples=100`? Because it would already be way more than enough